templates/repo/view_list.tmpl

```diff
@@ -30,7 +32,7 @@
 			</th>
 			<th class="nine wide">
 			</th>
-			<th class="three wide text grey right age">{{TimeSince .LatestCommit.Author.When $.Lang}}</th>
+			<th class="three wide text grey right age">{{if .LatestCommit.Author}}{{TimeSince .LatestCommit.Author.When $.Lang}}{{end}}</th>
```

Comment:
Missing an "else" here ?

Reply:
No, if there is no time than we just don't show anything

Reply:
or show `-` ?

Reply:
I don't think it's needed as it visually won't give any meaningful info